src/command_modules/azure-cli-shell/setup.py

```diff
@@ -35,6 +35,7 @@
     'applicationinsights',
     'azure-cli-core',
     'jmespath',
+    'humanfriendly',
```

Comment:
This should actually be added as a dep. to `azure-cli-core` as that's where it's used, not here.